DATA/production/qc-async/mft.json

```diff
@@ -39,8 +39,17 @@
           "maxClusterROFSize" : "50000",
           "maxDuration" : "60000",
           "timeBinSize" : "0.1",
-          "ROFLengthInBC" : "594",
-          "geomFileName" : "o2sim_geometry-aligned.root"
+          "ROFLengthInBC" : "594"
+        },
+	"grpGeomRequest" : {
+          "geomRequest": "Aligned",
+          "askGRPECS": "false",
+          "askGRPLHCIF": "false",
+          "askGRPMagField": "true",
+          "askMatLUT": "false",
+          "askTime": "false",
+          "askOnceAllButField": "false",
+          "needPropagatorD":  "false"
```

Comment:
I assume that the Geometry Request which is here, should be added also to:
- https://github.com/AliceO2Group/O2DPG/blob/master/DATA/production/qc-sync/mft-full.json
- https://github.com/AliceO2Group/O2DPG/blob/master/DATA/production/qc-sync/mft-full-no-tracks.json
- https://github.com/AliceO2Group/O2DPG/blob/master/MC/config/QC/json/mft-clusters.json
I will try to do a local test but I need to rebuild as I have old versions of the SW,